win32/WinController.cpp

```diff
@@ -1,5 +1,5 @@
 /*
-* Descent 3 
```

Comment:
outside of the context of this PR but we should totally merge WinController and lnxcontroller if we can